src/test/java/marquez/common/models/CommonModelGenerator.java

```diff
@@ -92,8 +92,12 @@ public static DbName newDbNameWith(String value) {
   }
 
   public static ConnectionUrl newConnectionUrl() {
+    return newConnectionUrlWithDatasourceType(newDatasourceType());
+  }
+
+  public static ConnectionUrl newConnectionUrlWithDatasourceType(DatasourceType datasourceType) {
```

Comment:
_Minor:_ The naming convention we have for generators is `newClazzWith(Overrides...)`, where `Clazz` is the class you'd like to instantiate and `Overrides...` are any overrides you'd like to provide.

So a generator method for `ConnectionUrl` that overrides the `DatasourceType` would be:

```java
newConnectionUrlWith(DatasourceType datasourceType)
```

Reply:
Ok.